govcd/api_vcd.go

```diff
@@ -105,7 +105,7 @@ func NewVCDClient(vcdEndpoint url.URL, insecure bool, options ...VCDClientOption
 	// Setting defaults
 	vcdClient := &VCDClient{
 		Client: Client{
-			APIVersion: "33.0", // supported by 10.0+
+			APIVersion: "34.0", // supported by 10.1+
```

Comment:
I propose adding the possibility of changing this value using an environment variable (`GOVCD_API_VERSION`). This way, we could change the version on-the-fly for debugging or when dealing with VCD upgrades that don't support our minimum API version.

Reply:
@lvirbalas @adambarreiro @vbauzysvmware @Didainius what do you think about this?

Reply:
This would be useful in some cases. We still leave the default, but just have an option to alter it on the fly.

Reply:
@lvirbalas @Didainius @dataclouder @adambarreiro please check the new feature that supports what Giuseppe suggested